config/core/test-configs.yaml

```diff
@@ -1974,6 +1974,9 @@ test_configs:
   - device_type: meson-gxl-s905x-libretech-cc
     test_plans:
       - baseline
+      - kselftest-filesystems
```

Comment:
The subject of this commit is wrong too, it's about the device types and not the lab config.  So this would seem better: `test-configs.yaml: enable kselftest on some libretech devices`

You can then mention the fact that lab-baylibre has those devices in the commit message.